README.md

```diff
@@ -2,140 +2,95 @@
-- ![TypeScript](https://img.shields.io/badge/TypeScript-3178C6?logo=typescript&logoColor=white)
-- ![Express](https://img.shields.io/badge/Express-black?logo=express&logoColor=white)
-- ![PostgreSQL](https://img.shields.io/badge/PostgresSQL-4169E1?logo=postgresql&logoColor=white)
-- ![Prisma](https://img.shields.io/badge/Prisma-white?logo=prisma&logoColor=2D3748)
-- ![Heroku](https://img.shields.io/badge/Heroku-430098?logo=heroku&logoColor=white)
+<table>
+   <tbody>
+      <tr>
+         <td align="center" valign="top" width="14.28%"><a href="https://github.com/Anissa3005"><img src="https://avatars.githubusercontent.com/u/114712265?v=4" width="100px;" alt="Anissa Chadouli"/><br /><sub><b>Anissa Chadouli</b></sub></a><br /><a href="https://github.com/Anissa3005" title="Profile">💻</a></td>
+         <td align="center" valign="top" width="14.28%"><a href="https://github.com/FefesG5"><img src="https://avatars.githubusercontent.com/u/116435020?v=4" width="100px;" alt="Gee Chai"/><br /><sub><b>Gee Chai</b></sub></a><br /><a href="https://github.com/FefesG5" title="Profile">💻</a></td>
+         <td align="center" valign="top" width="14.28%"><a href="https://github.com/shonancanuck"><img src="https://avatars.githubusercontent.com/u/105531674?v=4" width="100px;" alt="Sean McKay"/><br /><sub><b>Sean McKay</b></sub></a><br /><a href="https://github.com/shonancanuck" title="Profile">💻</a></td>
+         <td align="center" valign="top" width="14.28%"><a href="https://github.com/oddpharmacy"><img src="https://avatars.githubusercontent.com/u/107352832?v=4" width="100px;" alt="Lisa Tan"/><br /><sub><b>Lisa Tan</b></sub></a><br /><a href="https://github.com/oddpharmacy" title="Profile">💻</a></td>
+      </tr>
+   </tbody>
+</table>
 
 ## Getting Started
 
-### Prerequisites
+### ✅ Prerequisites
 
-To be able to use this application you have to make use of PostgresSQL version 12.14 or higher, and Node.js version 16.8 or later.
+- [Node](https://nodejs.org/en/) version 18.18.2
+- [Docker Dektop](https://www.docker.com/products/docker-desktop/)
 
-You can check your version by running the following commands in your terminal:
+### 🛠️ Setup
 
-```sh
-node --version
-```
+#### 1. Install dependencies
 
 ```sh
-postgres --version
+npm install
 ```
 
-### Installation
+#### 2. Setup Husky hooks 🐶
 
-_Below you will find the basic installation and set up._
-
-1. Clone the repo
+This will run linting before pushing your commits.
 
 ```sh
-git clone https://github.com/your_username_/Project-Name.git
+npm run prepare
 ```
 
-2. Module Installation
+#### 3. Setup Database 🐳
 
-```sh
-npm install
-```
+To keep in simple we use Docker to run an image of PostgreSQL database.
 
-3. Creating a database with Postgres
+#### 3.a Add database URL to your .env file
 
-- Mac:
+This should match with the Docker environment.
 
 ```sh
-psql postgres
+DATABASE_URL=postgresql://dbuser:passworddb@localhost:5432/namedb
```

Comment:
This is the url for a local instance. We can probably provide the url for the docker instance?